Source/CarthageKit/LocalRepository.swift

```diff
@@ -0,0 +1,238 @@
+// swiftlint:disable vertical_parameter_alignment_on_call
+import Foundation
+import Tentacle
+import Result
+import ReactiveSwift
+
+/**
+Local repository for storing/loading dependencies and their versions. This is for testing without requiring live connection to real repositories.
+*/
+public final class LocalRepository {
```

Comment:
The name is a bit confusing to me as it overlaps very much with git repository.

If I understand correctly this is just is just a wrapper around a URL for where to get the serialized dependency information from.

Maybe a name like `ResolverDiagnsticStore` suits this better.

Reply:
Renamed it to LocalDependencyStore